src/WebJobs.Script.WebHost/App_Start/WebApiConfig.cs

```diff
@@ -114,10 +114,11 @@ private static WebHostSettings GetDefaultSettings()
             WebHostSettings settings = new WebHostSettings();
 
             string home = Environment.GetEnvironmentVariable("HOME");
+            string scriptPath = Environment.GetEnvironmentVariable("SCRIPTPATH");
```

Comment:
The WebJobs.Script.Host already has a pattern for this. In your local project settings, you can point to whatever script dir you want. F5 the host and it runs with those scripts. I think that will meet your needs? This is explained in the wiki https://github.com/Azure/azure-webjobs-sdk-script/wiki (look for ..\..\..\..\sample).